src/build/packages.rs

```diff
@@ -142,6 +154,10 @@ pub fn read_folders(
 
     for entry in fs::read_dir(package_dir.join(&path_buf))? {
         let entry_path_buf = entry.map(|entry| entry.path())?;
+        if is_broken_symlink(&entry_path_buf) {
+            println!("Warning: '{}' is a broken symlink", entry_path_buf.display());
+            continue;
+        }
```

Comment:
Just an idea, but could this check be moved to if an actual error is produced? I assume it'd be in the line below, `let metadata = fs::metadata(&entry_path_buf)?;`.

Maybe it doesn't matter in practice, but sometimes these FS APIs incur latency, so it'd be nice to avoid using it unless actually needed, which I guess is only if it actually errors.

Reply:
Yeah - +1 @adnelson -- I think push these to both 169, as well as 471. I think both those could potentially be symlinked folders. Other than that, great stuff.

It reminds me that I should probably pick up the PR to add proper log levels again at some point.

Reply:
@adnelson -- We've added some better loggin structure, feel free to rebase and update the PR.

If you don't feel like it - I'd be happy to take this over and finish it, I think this could be nice to have :)